File: Femis Assignments/Morning Snacks/Armstrong.py
number = int(input("Enter an integer: "))
actualNumber = number

digits = len(str(number))

sum_of_digits = 0

while actualNumber > 0:
    digit = actualNumber % 10

    power = 1
    for i in range(digits):
        power *= digit

    sum_of_digits += power
    actualNumber //= 10

if sum_of_digits == number:
    print(number, "is an Armstrong number")
else:
    print(number, "is NOT an Armstrong number")

File: Femis Assignments/Morning Snacks/Armstrong.java
import java.util.Scanner;
public class Armstrong {
public static void main(String[] args) {
Scanner input = new Scanner(System.in);
System.out.println("Enter an integer");
int number = input.nextInt();
int actualNumber = number;
int digits = String.valueOf(number).length();
int sum = 0;

while (actualNumber > 0) {
    int digit = actualNumber % 10;
    sum += Math.pow(digit, digits);
    actualNumber /= 10;
}

if (sum == number) {
    System.out.println(number + " is an Armstrong number");
} else {
    System.out.println(number + " is NOT an Armstrong number");
}
}
}

